libyui/src/YDialogSpy.cc

```diff
@@ -626,73 +653,86 @@ void YDialogSpyPrivate::deleteWidget()
     }
 
     // any other child left after the removal?
-    if (!parent->hasChildren())
+    if ( ! parent->hasChildren() )
     {
 	// add an Empty widget to have a valid widget tree
 	// e.g. empty VBoxes are not allowed
-	YUI::widgetFactory()->createEmpty(parent);
+	YUI::widgetFactory()->createEmpty( parent );
     }
 
     targetDialogUpdated();
 }
 
 /**
  * Helper method - Is the widget a VBox or Hbox?
+ *
  * @param  widget the widget
- * @return	  true if the widget is a VBox or HBox
+ * @return true if the widget is a VBox or HBox
  */
 bool isBox(const YWidget *widget)
 {
-    return dynamic_cast<const YLayoutBox *>(widget);
+    return dynamic_cast<const YLayoutBox *>( widget );
 }
 
+
 /**
  * Helper method - Is the widget a VBox?
+ *
  * @param  widget the widget
- * @return	  true if the widget is a VBox
+ * @return true if the widget is a VBox
  */
-bool isVBox(const YWidget *widget)
+bool isVBox( const YWidget *widget )
 {
-    auto box = dynamic_cast<const YLayoutBox *>(widget);
+    auto box = dynamic_cast<const YLayoutBox *>( widget );
+
     return box && box->primary() == YD_VERT;
 }
 
+
 /**
  * Move the selected widget up/left or down/right. The visual direction
  * actually depends on the widget, it just moves the widget to the begining
  * or the end of the container.
+ *
  * @param true = up move to the begining (up/left), false = to the end (down/right)
  */
-void YDialogSpyPrivate::moveSelected(Direction direction)
+void YDialogSpyPrivate::moveSelected( Direction direction )
 {
     auto target_widget = selectedWidget();
-    if (!target_widget) return;
+
+    if ( !target_widget )
```

Comment:
We mandate a space after unary operators?

Reply:
Right; fixing.

Reply:
There is probably still cruft left over in any of those files. It will get less over time.